theme.js

```diff
@@ -1,3 +1,110 @@
+// Add dark theme support
+(function() {
+    // Check if dark mode is enabled in the store
+    const isDarkMode = store.get("dark-mode") === true;
+    
+    // Apply dark-theme class if dark mode is enabled
+    if (isDarkMode) {
+        document.body.classList.add('dark-theme');
+    } else {
+        document.body.classList.remove('dark-theme');
+    }
+    
+    // Update theme when dark mode setting changes
+    ipc.on('theme-changed', function(event, isDark) {
+        if (isDark) {
+            document.body.classList.add('dark-theme');
+        } else {
+            document.body.classList.remove('dark-theme');
+        }
+    });
+})();
+
+// Apply custom colors to dropdown elements
+$('body').append(
```

Comment:
If you search in all code base you will find that dark mode is already supported in somewhere else so you shall not implement it again.